yb-voyager/cmd/importData.go

```diff
@@ -276,6 +305,12 @@ func importData() {
 		targetUriList = append(targetUriList, t.Uri)
 	}
 	log.Infof("targetUriList: %s", utils.GetRedactedURLs(targetUriList))
+
+	if parallelImportJobs == -1 {
+		parallelImportJobs = fetchDefaultParllelJobs(targets)
+		utils.PrintAndLog("Using %d parallel jobs by default. Use --parallel-jobs to specify a custom value", parallelImportJobs)
```

Comment:
Take this call out of the `if` block. That will ensure that the message is show even for LB cases.

Reply:
If you are talking about the call to the print message, I only show this if parallel-jobs is not provided explicitly, which is why I have added two separate instances of print messages for the LB case and the other generic case